tests/cmds/test_users.py

```diff
@@ -388,3 +398,26 @@ def test_bulk_deactivate_uses_expected_arguments_when_all_are_passed(
             "updated": "False",
         }
     ]
+
+
+def test_change_org_calls_change_org_assignment_with_correct_parameters(
+    runner, cli_state, change_org_success, get_user_id_success
+):
+    command = ["users", "move", "--username", TEST_USERNAME, "--org-id", "54321"]
+    runner.invoke(cli, command, obj=cli_state)
+    cli_state.sdk.users.change_org_assignment.assert_called_once_with(
+        user_id=TEST_USER_ID, org_id=54321
+    )
+
+
+def test_bulk_move_uses_expected_arguments(runner, mocker, cli_state):
+    bulk_processor = mocker.patch(f"{_NAMESPACE}.run_bulk_process")
+    with runner.isolated_filesystem():
+        with open("test_bulk_move.csv", "w") as csv:
+            csv.writelines(["username,org_id\n", f"{TEST_USERNAME},4321\n"])
+        runner.invoke(
+            cli, ["users", "bulk", "move", "test_bulk_move.csv"], obj=cli_state
+        )
+    assert bulk_processor.call_args[0][1] == [
```

Comment:
If this is just called once, you can use `assert_called_once_with()` and pass in the args and kwargs (the same as they get passed in the real call).

It helps ensure the method was also called the expected number of times.

Reply:
What we're testing her is the call to `run_bulk_process`, which takes as its argument the `handle_row` method, the csv rows themselves, and a progress label. The test is only testing that the csv rows are passed through as expected, which is the same approach we take with testing the other bulk methods.

I've added another assertion to this test to verify that it's only called once.

Reply:
Oh I see, it is because that method-call contains like a method and what-not.
Thanks, `assert_called_once()` is probably best here.